flathunter/chrome_wrapper.py

```diff
@@ -56,6 +57,8 @@ def get_chrome_driver(driver_arguments):
     """Configure Chrome WebDriver"""
     logger.info('Initializing Chrome WebDriver for crawler...')
     chrome_options = uc.ChromeOptions() # pylint: disable=no-member
+    if platform == "darwin":
+            chrome_options.add_argument("--headless")
```

Comment:
What happens if `--headless` is already in `driver_arguments`? Does it get added twice? Is that a problem?

Reply:
I'm not sure - without adding `--headless` here `headless` option in `driver_arguments` didn't worked because of error "AttributeError: 'ChromeOptions' object has no attribute 'headless'" (macOS)

Reply:
> What happens if `--headless` is already in `driver_arguments`? Does it get added twice? Is that a problem?

I'm not sure - without adding `--headless` here `headless` option in `driver_arguments` didn't worked because of error "AttributeError: 'ChromeOptions' object has no attribute 'headless'" (macOS)

Reply:
And adding `--headless` in `driver_arguments` (instead of `headless`)?

Reply:
Sorry, a typo, "--headless", of course:
```
driver_arguments:
        - "--no-sandbox"
        - "--headless"
        - "--disable-gpu"
        - "--remote-debugging-port=9222"
        - "--disable-dev-shm-usage"
        - "window-size=1024,768"
```

Reply:
Okay. That's very strange. But if it's making your life better then we can merge it. I'll just let the tests run again.

Reply:
Really strange! I used the same config on Linux and didn't have this problem